numpyro/infer/svi.py

```diff
@@ -101,6 +101,8 @@ class SVI(object):
         >>> svi = SVI(model, guide, optimizer, loss=Trace_ELBO())
         >>> svi_result = svi.run(random.PRNGKey(0), 2000, data)
         >>> params = svi_result.params
+        >>> predictive = Predictive(model, guide=guide, params=params, num_samples=1000)
+        >>> samples = predictive(random.PRNGKey(0), data)
```

Comment:
This code does not return any useful information unless you set `data=None` (which will give you posterior predictive samples). If you want to get posterior samples, then you can move those codes to the end and add a comment
```python
>>> # get posterior samples
>>> predictive = Predictive(guide, params=params, num_samples=1000)
>>> samples = predictive(random.PRNGKey(1), data)
```